src/scss/_header.scss

```
@@ -40,6 +40,54 @@
             }
         }
     }
+
+    // The nav bar will be pinned on mobile but not on desktop.
+    &.registry {
+        @apply sticky z-50 bg-gray-850;
+        top: -1px;
```

Believe it or not...

```suggestion
        @apply sticky z-50 bg-gray-850 -top-px;
```